news/12345-bugfix-for-list-all-known-prefixes

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Filter out None path values from pwd.getpwall() on Unix systems, for users without home directories, when running as root. (#12063)
```

Comment:
These are probably system users, if you `cat /etc/passwd` there are a ton of users for different programs.